DictionaryTypes.txt

```diff
@@ -452,8 +452,10 @@ enum(sex) "Sex of a subject/patient"
 
 enum(hand) "Handedness of a subject/patient"
 {
+    n/a   0  "not available"
```

Comment:
I don't think we need this one, `hand` is optional (I think based on the `?` at the end of the line):

https://github.com/mne-tools/fiff-constants/blob/6733c5eb8a3e036c83ab10915de487fe77016546/DictionaryStructures.txt#L181

In MNE-Python this shows up as the key just not being in `info['subject_info']`.

Reply:
ok, i can remove it.. 

the same applies to sex, though

plus practically, it might make sense to have an explicit n/a option (e.g. if there is a study with several participants, and the info is only missing for one or two of them)

Reply:
> plus practically, it might make sense to have an explicit n/a option (e.g. if there is a study with several participants, and the info is only missing for one or two of them)

I expect that the difference should be fairly small, for example you would just do `hand = info['subject_info'].get('hand', 'n/a')` instead of just `info['subject_info']['hand']`. I think the `?` implies that the "FIFF constant way" of having a "n/a" entry is just not to write it, so it seems more repo-consistent not to add this option to the spec (and even it seems more MNE-consistent, since you will already need such checks for other things like `info['subject_info']['weight']`, etc.).

Reply:
I would let the value missing in file and at none in python. I agree with @larsoner

Reply:
alright, i removed the option

Reply:
..and thanks for the explanation @larsoner !